### DISCUSSION TITLE
有一个问题

### DISCUSSION THREAD

**Discussion by lzqv875** [hhyo/Archery#2519] [General]
版本v.1.10.0中docker-compose.yml中配置错误，导致安装v.1.10.0版本执行python3 manage.py makemigrations sql失败
![屏幕截图 2024-02-22 185138](https://github.com/hhyo/Archery/assets/107014798/bb3b4c5b-992d-4056-ade6-027edcafc1c7)

**Comment by LeoQuote** [comment]
什么问题呢？提个pr呗
